packages/create-cloudflare/src/wrangler/__tests__/config.test.ts

```diff
@@ -83,6 +83,12 @@ describe("update wrangler config", () => {
 			# [[services]]
 			# binding = "MY_SERVICE"
 			# service = "my-service"
+
+			# KV Namespaces
```

Comment:
shouldn't this be under the bindings section above? (line 61) 🤔

Reply:
oh...but I see now what u mean...I'll fix